### PR TITLE
feat: completed login and signup page

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
image
image

### CODE REVIEW

**Review by Namrata-J — APPROVED** [review]
code looks clean and is well structured

**Review by harshalini — APPROVED** [review]
Well structured code, good to merge

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { Cart } from "./pages/cartPage/cartPage.js";
 import { Wishlist } from "./pages/wishlistPage/wishlistPage.js";
 import { Home } from "./pages/homePage/home.js";
+import {Login} from './pages/authPages/login.js'
+import {Signup} from './pages/authPages/signup.js'
```
Comment (t007rushi) [inline src/App.js:5]: Barrel export it

File: src/components/auth/signup.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            <button className="forget-pwd-btn">forget password</button>
+          </div>
+          <button type="submit" className="btn-outline form-btn">
+          Signup
+          </button>
+
+          <aside className="flex-wrap">
+            <span>Already have an account &nbsp;</span>
+            <Link to="/profile" className="link-tag">
+              Login&nbsp;{`>>`}
```
Comment (t007rushi) [inline src/components/auth/signup.js:69]: Use icons instead

**Review by t007rushi — APPROVED** [review]
Go ahead with some suggestions

**Pull request merged**